packages/app/app/components/AlbumView/index.tsx

```diff
@@ -138,7 +138,7 @@ export const AlbumView: React.FC<AlbumViewProps> = ({
                   <th className={cx(styles.center, styles.position)}>
                     <Icon name='hashtag' />
                   </th>
-                  <th className={styles.left}>Song</th>
+                  <th className={styles.left}>Tracks</th>
```

Comment:
can you use locale in stead of this hard string

Reply:
Sorry, what did you mean with locale?

Reply:
I changed it to how it was originally

Reply:
These need to be translated according to the selected language, using `react-i18next`.

Reply:
you can see in the same file that:
```nodejs
const { t } = useTranslation('album');
...
<PopupButton
    onClick={addAlbumToDownloads}
    ariaLabel={t('download')}
    icon='download'
    label={t('download')}
  />
```
and the `download` key is defined in `packages/i18n/src/locales/en.json` as:
```js
{
  ...
  album: {
     ...
     download: 'Download'
  }
}
```

Reply:
Hi! I changed it again using `react-i18next` I was only able to translate to Spanish, the others are in English :)

Reply:
That good 😌